dask/dataframe/io/tests/test_io.py

```diff
@@ -139,14 +139,7 @@ def check():
         assert (sorted(dd.from_bcolz(t, chunksize=2).dask) !=
                 sorted(dd.from_bcolz(t, chunksize=3).dask))
 
-    threads = []
```

Comment:
FYI @mrocklin, this change was because the old style failed to ever raise an exception for me, even with

```python
def check():
    assert 0
```

The exception would happen in the other thread, but would fail to cause an exception in the main test process.